exercises/anagram/src/example.c

```diff
@@ -5,56 +5,39 @@
 
 #include "anagram.h"
 
-char *lower_case(char *p)
+void to_lower_case(char p[])
```

Comment:
This can be `static`, right?

Reply:
While this is being changed anyway, `str` might be more idiomatic than `p` for an argument to a function that does basic operations on a string (see string.h for example)